docs/VnVPlan/VnVPlan.tex

```diff
@@ -782,6 +783,87 @@ \subsubsection{Work Orders}
 
 \end{enumerate}
 
+\subsubsection{Services}
+
+This section will contains tests covering the ``Services'' requirements defined in the
+\href{https://github.com/arkinmodi/project-sayyara/blob/main/docs/SRS/SRS.pdf}{SRS}.
+
+\begin{enumerate}
+
+	\item \textbf{FRT-BE28-1}
+
+	      Control: Manual
+
+	      Initial State: Shop Owner has a shop profile
+
+	      Input: Shop Owner sets available services
+
+	      Output: Shop services are sent to the database and displayed on the shop profile
+
+	      Test Case Derivation: The user can add available auto shop services to their shop profile
+
+	      How test will be performed: The user will search for shop services through the user interface
+
+	\item \textbf{FRT-BE29-1}
+
+	      Control: Manual
+
+	      Initial State: Customer Account searches for auto repair or maintenance services
+
+	      Input: Enters specific auto service in the search bar
+
+	      Output: Auto shops with relevant services displayed with service details
+
+	      Test Case Derivation: Services and auto shops relevant to the search query will be displayed
+
+	      How test will be performed: The user will search for shop services through the user interface
+
+	\item \textbf{FRT-BE29-2}
+
+	      Control: Manual
+
+	      Initial State: Shop Owner/Shop Employee searches for auto repair or maintenance services
+
+	      Input: Enters specific auto service in the search bar
+
+	      Output: Auto shops with relevant services displayed with service details
```

Comment:
What do we plan to be different between the customer's output vs the shop owner/employee's out for BE29?

Reply:
The SRS does not really make this clear. Maybe we should have a follow up to update the SRS.